qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -1655,7 +1655,7 @@ def Triangle(
 
     .. math::
 
-        f(x) &= \\text{A}\\left[\\text{sawtooth}\\left(x\\right)right]  ,  0 <= x < duration
+        f(x) = \\text{A}\\left[\\text{sawtooth}\\left(x\\right)right]  ,  0 <= x < duration
```

Comment:
```suggestion
        f(x) = \\text{A}\\left[\\text{sawtooth}\\left(x\\right)\\right]  ,  0 <= x < duration
```

this looks like the only remnant mistake